hack/make/test-unit

```diff
@@ -48,8 +48,8 @@ bundle_test_unit() {
 			| grep -v github.com/docker/docker/man \
 			| grep -v github.com/docker/docker/integration-cli)
 	fi
-
 	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS $pkg_list
+	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS github.com/docker/docker/pkg/loopback -test.root
```

Comment:
Yes I have already done that as you can see in the test file in the function `RequiresRoot(t *testing.T)`. If you provide the flag and there is no root access the test case fails. You have to provide both the flag and root access for these test cases to work.